gravity.sh

```diff
@@ -43,7 +45,7 @@ else
 fi
 
 #Remove the /* from the end of the IPv4addr.
-IPv4addr=${IPv4_address%/*}
+IPv4_address=${IPv4_address%/*}
```

Comment:
The value should be stored in `setupVars` with the CIDR notation, as that file will be sourced for headless installation and we need the subnet information. I don't believe it's used in CIDR form within `gravity.sh`, just plain as the source address for the hosts files. If you want to break out `setupVars` and store the CIDR value as a separate variable, then there's that option, but we do need to have it stored for access.